src/Widgets/Presets/DefaultOrderConfirmationPreset.php

```diff
@@ -86,80 +99,105 @@ private function createHeadline()
+                                ->withSetting("block", "true")
+                                ->withSetting("text", $this->translator->trans("Ceres::Widget.urlReturnLabel"))
+                                ->withSetting("url.value", "return")
+                                ->withSetting("url.type", "internalLink");
     }
 
     private function createOrderTotalsWidget()
     {
         $this->twoColumnWidget->createChild("second", "Ceres::OrderTotalsWidget")
-                              ->withSetting("spacing.customMargin", true)
-                              ->withSetting("spacing.margin.top.value", 3)
-                              ->withSetting("spacing.margin.top.unit", null)
-                              ->withSetting("spacing.margin.left.value", 4)
-                              ->withSetting("spacing.margin.left.unit", null)
-                              ->withSetting("spacing.margin.right.value", 4)
-                              ->withSetting("spacing.margin.right.unit", null)
                               ->withSetting("visibleFields", ["orderValueNet", "orderValueGross", "rebate", "shippingCostsNet", "shippingCostsGross", "totalSumNet", "promotionCoupon", "vats", "totalSumGross", "salesCoupon", "openAmount"]);
     }
 
-    private function createHomeLinkWidget()
+    private function createSeparatorWidget()
     {
-        $this->fourColumnWidget->createChild("second", "Ceres::LinkWidget");
+        $this->preset->createWidget("Ceres::SeparatorWidget")
+                     ->withSetting("customClass","");
     }
 
-    private function createMyAccountLinkWidget()
+    private function createFourColumnWidget()
     {
-        $this->fourColumnWidget->createChild("third", "Ceres::LinkWidget");
+        $this->fourColumnWidget = $this->preset->createWidget("Ceres::FourColumnWidget")
+                                               ->withSetting("customClass","");
+    }
+
+    private function createBottomNavigation()
+    {
+        $homepageLinkWidget = null;
+        $homepageLinkWidget = $this->fourColumnWidget->createChild("second", "Ceres::LinkWidget")
+                                        ->withSetting("appearance", "primary")
+                                        ->withSetting("block", "true")
+                                        ->withSetting("text", $this->translator->trans("Ceres::Template.orderConfirmationHomepage"));
+
+        $homepageLinkWidget->withSetting("url.type", "external")
+                            ->withSetting("url.value", $this->shopUrls->home);
+
+        if ( in_array(RouteConfig::HOME, RouteConfig::getEnabledRoutes()) && RouteConfig::getCategoryId(RouteConfig::HOME) > 0 )
+        {
+            $homepageLinkWidget->withSetting("url.type", "category")
+                               ->withSetting("url.value", RouteConfig::getCategoryId(RouteConfig::HOME));
+        }
+        else
+        {
+            $homepageLinkWidget->withSetting("url.type", "external")
```

Comment:
Was steht in ShopUrls::Home? `/`? => Macht das hier Sinn? Die Homepage sollte immer auf `/` liegen...

Reply:
Kann so bleiben